routing/router.go

```diff
@@ -1155,10 +1189,12 @@ func (r *ChannelRouter) processUpdate(msg interface{}) error {
 			if err := r.cfg.Graph.AddChannelEdge(msg); err != nil {
 				return fmt.Errorf("unable to add edge: %v", err)
 			}
-			log.Infof("New channel discovered! Link "+
+			log.Tracef("New channel discovered! Link "+
```

Comment:
Maybe a bit aggressive to demote to `Trace`? I find these logs useful sometimes running itests, where we use debug level.

Reply:
hmm my hope was that we could get rid of these at the `Debug` level since they're superrr spammy and aren't that useful in production. I run my node with `CRTR=debug` so just moving them to debug wouldn't change anything.